core/tests/protractor/learnerDashboard.js

```diff
@@ -91,9 +95,9 @@ describe('Learner dashboard functionality', function() {
     player.submitAnswer(
       'MultipleChoiceInput', 'Those were all the questions I had!');
     player.submitAnswer('Continue', null);
-    browser.get(general.LEARNER_DASHBOARD_URL);
+    learnerDashboardPage.get();
     browser.waitForAngular();
-    element(by.css('.protractor-test-completed-section')).click();
+    learnerDashboardPage.navigateToCompletedSection();
```

Comment:
Just to check -- is this completed collections, completed explorations, or something else?

Reply:
I think its a section of completed explorations and collections. In learnerDashboard.js, it is called above the subsection that calls both of them.